api.md

```diff
@@ -3618,35 +3619,46 @@ information as a separate row per node.
 #### Returns [](hypertable_detailed_size-returns)
 |Column|Description|
 |---|---|
-|table_bytes|(BIGINT) Disk space used by main_table (like pg_relation_size(main_table))|
-|index_bytes|(BIGINT) Disk space used by indexes|
-|toast_bytes|(BIGINT) Disk space of toast tables|
-|total_bytes|(BIGINT) Total disk space used by the specified table, including all indexes and TOAST data|
-|node_name| (NAME) Node for which size is reported, applicable only to distributed hypertables|
+|`table_bytes`|(BIGINT) Disk space used by main_table (like pg_relation_size(main_table))|
+|`index_bytes`|(BIGINT) Disk space used by indexes|
+|`toast_bytes`|(BIGINT) Disk space of toast tables|
+|`total_bytes`|(BIGINT) Total disk space used by the specified table, including all indexes and TOAST data|
+|`node_name`| (NAME) Node for which size is reported, applicable only to distributed hypertables|
```

Comment:
Should note that this includes size on the access node and node_name is displayed as NULL (for the access node).